todo-list/components/layout.js

```diff
@@ -0,0 +1,69 @@
+import Head from 'next/head';
+import Image from 'next/image';
+import styles from './layout.module.css';
+import utilStyles from '../styles/utils.module.css';
+import Link from 'next/link';
+
+const name = 'Vasyl Luchka';
+export const siteTitle = 'Next.js Sample Website';
+
+export default function Layout({ children, home }) {
+  return (
+    <div className={styles.container}>
```

Comment:
Так як Head не буде в конкретному `div`, можна організувати таким чином:
```suggestion
    <>
      <Head>...</Head>
      <div className={styles.container}>...</div>
    </>

```